### DISCUSSION TITLE
Get STW codes

### DISCUSSION THREAD

**Discussion by mirronake** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#113] [Q&A] [answered]
Hello! How i'm can get STW codes ( STW Founder codes ) using API ?

**Comment by LeleDerGrasshalmi** [comment]
mg?
that feature is long unsued?

**Reply by LeleDerGrasshalmi** [reply, marked as answer]
GET `https://fngw-mcp-gc-livefn.ol.epicgames.com/fortnite/api/game/v2/friendcodes/:accountId/:codeType`
`codeType` can be `epic` or `xbox`

(This will be added to the repo today)

**Comment by Noma77** [comment]
I know it's old, but does anyone have a way to get a founder's key? I really don't have the money to have it

**Comment by matyx36** [comment]
how should i use that link to get the codes? can someone help me, i dont understand how to get the codes by api?
